packages/block-library/src/paragraph/edit.js

```diff
@@ -48,6 +56,27 @@ function ParagraphRTLToolbar( { direction, setDirection } ) {
 	) );
 }
 
+function useDropCapMinimumHeight( isDropCap ) {
+	const [ minimumHeight, setMinimumHeight ] = useState();
+	useEffect(
+		() => {
+			const element = querySelector && querySelector( PARAGRAPH_DROP_CAP_SELECTOR );
+			if ( isDropCap && element ) {
+				setMinimumHeight(
+					getComputedStyle(
+						element,
+						'first-letter'
+					).height
+				);
+			} else if ( minimumHeight ) {
```

Comment:
The closure over `minimumHeight` could be stale, because it's not a dep, leading to the minimum height not being cleared in some cases where it should.